### DISCUSSION TITLE
Change Station Location

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5390] [General]
Ubuntu 20.04LTS
AzuraCast Stable

I have limited space available on /var/azuracast and set up 1 of my 13 stations in /data/azuracast/stations/----
The location is set up but trying to start the station broadcast I get this error
Liquidsoap is not recognized as a server.; It may not be registered with Supervisor yet. Restart broadcast may help.
I can not restart the broadcast because it does not start.
file location is set up and I can play a songs from media list
I have not found any help fixing this
If I set up the radio stations in the default location everything works and have 5 stations in /var/azuracast/stations and 1 set up in /data/azuracast/stations
I've tried setting up the radio station ins the default location and creating a symbolic link to the media files on data/azuracast/stations/jazz but that didn't work.
I've read the docs, searched the Discussions and haven't found the answer 
I need help making this work!

**Comment by ItsMitchh** [comment]
Hi there, this may require a issue for us to look at properly.

When you create a issue, please include logs and as much detail as possible.

**Comment by ghost** [comment]
SC I found an answer that works for me
I fixed my issue by creating a symbolic link and made no other changes to the setup. 
ln -s /datapath/to/files stations_name
